core/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -101,6 +101,9 @@ public ActionRequestValidationException validate() {
             validationException = ValidateActions.addValidationError("illegal version value [" + version + "] for version type [" + versionType.name() + "]",
                     validationException);
         }
+        if (versionType == VersionType.FORCE) {
```

Comment:
isn't this a problem if coming from an update request with force? should we do something there to enable this?